src/problem3.js

```diff
@@ -1,6 +1,19 @@
 function problem3(number) {
-  var answer;
-  return answer;
+  return getClapNum(number);
+}
+
+function getClapNum(number) {
```

Comment:
취향이긴한데 개인적으로는 횟수이니깐 num 대신 cnt 가 더 직관적일 것 같네용 ㅎㅎ

그냥 `ClapNum` 이라 하면, "몇 번째" 손뼉인지 같은 느낌

```suggestion
function getClapCnt(number) {
```

`number` 을 쓰고싶다면

```suggestion
function getTheNumberOfClaps(number) {
```

이 더 잘 읽히지 않을까용